structopt-derive/src/lib.rs

```diff
@@ -404,6 +406,20 @@ fn gen_augmentation(fields: &[Field], app_var: &Ident) -> quote::Tokens {
     }}
 }
 
+/// Interpret the value of `*_raw` attributes as code and the rest as strings.
+fn gen_attr_call(key: &syn::Ident, val: &syn::Lit) -> quote::Tokens {
+    if let Lit::Str(ref val, _) = *val {
+        let key = key.as_ref();
+        if key.ends_with("_raw") {
+            let key = Ident::from(&key[..(key.len() - 4)]);
+            // Call method without quoting the string
+            let ts = syn::parse_token_trees(val).unwrap();
```

Comment:
I think something like `.expect("bad parameter to {}: the parameter must be valid rust code", key_with_raw)`